node-websocket-chat/idl/.gitignore

```diff
@@ -0,0 +1,16 @@
+.depend.*
+.shobj**
+ChatC.cpp
+ChatC.h
+ChatC.inl
+ChatS.cpp
+ChatS.h
+ChatTypeSupport.idl
+ChatTypeSupportC.cpp
+ChatTypeSupportC.h
+ChatTypeSupportC.inl
+ChatTypeSupportImpl.cpp
+ChatTypeSupportImpl.h
+ChatTypeSupportS.cpp
+ChatTypeSupportS.h
+*.dylib
```

Comment:
```suggestion
```
Not that it matters too much, but I like keeping platform-specific info out of subdirectory gitignores.